packages/react-component-library/src/components/Timeline/helpers/children.ts

```diff
@@ -3,11 +3,10 @@ import React from 'react'
 import { TimelineDaysProps } from '../TimelineDays'
 import { TimelineMonthsProps } from '../TimelineMonths'
 import { TimelineRowsProps } from '../TimelineRows'
-import { TimelineSideProps } from '../TimelineSide'
 import { TimelineTodayMarkerProps } from '../TimelineTodayMarker'
 import { TimelineWeeksProps } from '../TimelineWeeks'
 
-type TimelineRootChildrenType = React.ReactElement<TimelineSideProps>
+type TimelineRootChildrenType = React.ReactElement<never>
```

Comment:
Is `TimelineRootChildrenType` still doing something or can it be removed from `timelineChildrenType` below?